test/e2e/apimachinery/garbage_collector.go

```diff
@@ -571,10 +568,9 @@ var _ = SIGDescribe("Garbage collector", func() {
 	})
 
 	/*
-		    Testname: garbage-collector-delete-deployment-propagation-true
-		    Description: Ensure that if deleteOptions.PropagationPolicy is set to Orphan,
-			then deleting a Deployment should cause ReplicaSets created
-			by that Deployment to be orphaned.
+		Release : v1.9
+		Testname: Garbage Collector, delete deployment, propagation policy orphan
+		Description: Create a deployment with a replicaset. Once replicaset is created , delete the deployment  with deleteOptions.PropagationPolicy set to Orphan. Deleting the deployment MUST cause the replicaset created by the deployment to be orphaned.
```

Comment:
Should state the expect behavior of the Pods too.

Reply:
Test is only testing for RS. Pods can be tested too.

Reply:
My point is that we need to state what the expected behavior for the pods are. In other words make it clear what people should expect to see.

Reply:
Fixed.